File: C/ex1008.c
#include <stdio.h>

float salrio(int horas, float valor_hora);

int	main()
{
	int		id, hours;
	float	value_hour;

	scanf("%d %d %f", &id, &hours, &value_hour);
	printf("NUMBER = %d\nSALARY = U$ %.2fl\n", id, salario(hours, value_hour));
}


float salrio(int horas, float valor_hora)
{
	return (horas * valor_hora);
}

File: GO/ex1008.go
package main

import(
	"fmt"
)

func salario(hours int, value_hours float64) float64{
	return float64(hours) * value_hours
}

func main(){
	var id, hours int
	var value_hours float64
	fmt.Scanf("%d", &id)
	fmt.Scanf("%d", &hours)
	fmt.Scanf("%f", &value_hours)
	fmt.Printf("NUMBER = %d\nSALARY = U$ %.2f\n", id, salario(hours, value_hours))
}
